docs/README.md

```diff
@@ -1,29 +1,30 @@
-# User Guide
+# Duke User Guide
 
-## Features 
+// Update the title above to match the actual product name
 
-### Feature-ABC
+// Product screenshot goes here
 
-Description of the feature.
+// Product intro goes here
 
-### Feature-XYZ
+## Adding deadlines
 
-Description of the feature.
+// Describe the action and its outcome.
 
-## Usage
+// Give examples of usage
 
-### `Keyword` - Describe action
+Example: `keyword (optional arguments)`
 
-Describe the action and its outcome.
+// A description of the expected outcome goes here
```

Comment:
Nice use of comments to note things to do in the future!

## Tip:
Adding the keyword `TODO:` in front of comments could be useful as some IDEs will list them in the `Problems` tool window. This will help you remember which parts of the code you have marked as todos.

```java
// TODO: Update the title above to match the actual product name

// TODO: Product screenshot goes here

// TODO: Product intro goes here
```

### Example:
<img width="1188" alt="image" src="https://github.com/nus-cs2103-AY2324S2/ip/assets/44436551/45d92153-bfb3-4e37-b61b-31219f6416d2">